modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginAnalyticsCloudTest} from '../../fixtures/loginAnalyticsCloudTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import {syncAnalyticsCloud} from '../analytics-settings-web/utils/analyticsSettings';
+import {faroConfig} from './faro.config';
+
+export const test = mergeTests(loginAnalyticsCloudTest(), loginTest());
+
+test('renders workspace list', async ({page}) => {
+	await syncAnalyticsCloud(page);
+
+	await page.goto('https://www.google.com');
+
+	await page.evaluate((liferayConfig) => {
+		const aTag = document.createElement('a');
+
+		aTag.href = liferayConfig.environment.baseUrl;
+
+		aTag.click();
+	}, liferayConfig);
+
+	await page.getByText('Welcome to Liferay');
+
+	await page.waitForTimeout(10000);
+
+	await page.goto(faroConfig.environment.baseUrl);
+
+	await page
+		.getByRole('link', {
+			name: 'FARO-DEV-liferay Liferay Demo Enterprise Plan',
+		})
+		.click();
+
+	await page.getByRole('button', {name: 'Beryl Commerce'}).click();
+
+	await page.getByRole('link', {exact: true, name: 'Liferay DXP'}).click();
+
+	await page.getByRole('link', {exact: true, name: 'Pages'}).click();
+
+	await page.getByRole('button', {name: 'Last 30 days'}).click();
+
+	await page.getByRole('menuitem', {name: 'Last 24 hours'}).click();
+
+	await page
+		.getByRole('cell', {name: 'Home - Liferay DXP'})
+		.getByRole('link')
+		.click();
+
+	await page.getByRole('link', {name: 'Path'}).click();
+
+	await expect(page.getByText('https://www.goo...')).toBeVisible({
+		timeout: 100 * 1000,
+	});
+
+	await expect(page.getByText('1', {exact: true}).nth(1)).toBeVisible({
+		timeout: 100 * 1000,
+	});
+
+	await expect(page.getByText('1', {exact: true}).first()).toBeVisible({
+		timeout: 100 * 1000,
+	});
+
+	await expect(page.getByText('1', {exact: true}).nth(2)).toBeVisible({
+		timeout: 100 * 1000,
+	});
```

Comment:
Hey @marcosapmf, you need to generate an exit page to an external página of the DXP portal